system-tests/runtimes/file-transfer-provider/src/main/java/org/eclipse/edc/test/extension/api/FileTransferDataSinkFactory.java

```diff
@@ -45,9 +45,15 @@ public boolean canHandle(DataFlowRequest request) {
 
     @Override
     public @NotNull Result<Boolean> validate(DataFlowRequest request) {
-        return Result.success(true);
+        return Result.success(validateRequest(request).succeeded());
```

Comment:
I think this could be simplified to:
`return validateRequest(request).map(it -> true);`